clients/storage/lib/google_api/storage/v1/api/objects.ex

```diff
@@ -441,9 +441,9 @@ defmodule GoogleApi.Storage.V1.Api.Objects do
 
   - connection (GoogleApi.Storage.V1.Connection): Connection to server
   - bucket (String): Name of the bucket in which to store the new object. Overrides the provided object metadata&#39;s bucket value, if any.
-  - upload_type (String): Upload type. Must be \&quot;multipart\&quot;.
-  - metadata (Object): Object metadata
-  - data (String): The file to upload
+  - upload_type (String): Upload type. Must be \&quot;multipart\&quot;
+  - metadata (Object): 
```

Comment:
Why did these descriptions disappear?

Reply:
This was previously generated from a hand-modified OpenAPI json spec and now is using a generated OpenAPI json spec.

Will fix.

Reply:
This should now be fixed.